src/test/scala/edu/ie3/mobsim/model/ElectricVehicleSpec.scala

```diff
@@ -272,10 +275,93 @@ class ElectricVehicleSpec extends UnitSpec with ElectricVehicleTestData {
 
             evs should have size targetAmount
             evs.count(
-              _.isChargingAtHomePossible
+              _.chargingAtHomePossible
             ) shouldBe expectedAmountOfHomeCharging
         }
       }
     }
+
+    "An electricVehicle" should {
```

Comment:
I like that you're testing the copy methods here. It might seem trivial at first, but I think we can get some value here in the long term